arc/cli/commands/diff.py

```
@@ -44,30 +40,74 @@ async def _initialize_state():
+        if run['config_path'].endswith(config_name) or run['config_path'] == identifier:
+            return run['run_id'], config_name
+    
+    return None, config_name
+
+
+def _find_common_scenarios(run1, run2) -> List[str]:
+    """Find scenarios that appear in both runs.
+    
+    Returns list of scenario IDs that are in both runs.
+    """
+    scenarios1 = {s.get('scenario_id', s.get('id', f'scenario_{i}')) 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)